docs/apm/using-the-apm-ui.asciidoc

```diff
@@ -27,14 +28,54 @@ You can add services by setting the `service.name` configuration in each of the
 [role="screenshot"]
 image::apm/images/apm-services-overview.png[Example view of services table the APM UI in Kibana]
 
+[[traces]]
+=== Traces
+
+The Traces overview page displays every *transaction* occurring in your application,
```

Comment:
I don't think this statement is entirely true, at least perhaps not the way it was intended. If you're running a non-distributed setup, then yes, it will consist of all transactions, but in a distributed setup we're listing the _entry transaction_ for all traces. If you're using distributed tracing, this list is key to finding that critical path in your application. @makwarth do you agree the perhaps phrasing here is a little off?

Reply:
I'm going to approve the rest of the changes, but pending @makwarth's input here.